style/ember/README.md

```diff
@@ -2,6 +2,7 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+* Don't use {{partial}} helper. Use a component with only a template.
```

Comment:
Since views have been depreciated is it worth adding them? If so it should certainly be a `don't` for using views?